src/swarm/util/log/ClientStats.d

```diff
@@ -146,6 +152,40 @@ public class ClientStats : StatsLog
     }
 
 
+    /***************************************************************************
+
+        Construct the class to use the default StatsLog appender, registers it
```

Comment:
Maybe use `Constructs` to be consistent with the tense in the rest of the sentence?